auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -203,7 +201,7 @@ export default class AffiliatedEntityTable extends Mixins(DateMixin) {
   // Local Properties
   private readonly businesses!: Business[]
   private readonly currentOrganization!: Organization
-  private readonly createNamedBusiness!: (filingBody: BusinessRequest, nrNumber: string) => Promise<any>
+  private readonly createNamedBusiness!: ({ filingType, business }) => Promise<any>
```

Comment:
As I mentioned above, only a single formal parameter can be passed to an Action. So the old never worked -- nrNumber was always undefined in the method. Fortunately it wasn't used there...

Note that createNamedBusiness now looks similar to createNumberedBusiness :)